manifests/2.19.0/opensearch-2.19.0-test.yml

```diff
@@ -0,0 +1,148 @@
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: flow-framework
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: asynchronous-search
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: cross-cluster-replication
+    integ-test:
+      topology:
+        - cluster_name: leader
+          data_nodes: 2
+        - cluster_name: follower
+          data_nodes: 2
+      test-configs:
+        - with-security
+        - without-security
+  - name: geospatial
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: index-management
+    integ-test:
+      build-dependencies:
+        - job-scheduler
+      test-configs:
+        - with-security
+        - without-security
+      additional-cluster-configs:
+        path.repo:
+          - /tmp
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: k-NN
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: ml-commons
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: neural-search
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: notifications
+    working-directory: notifications
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: opensearch-observability
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: opensearch-reports
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: security
+    integ-test:
+      test-configs:
+        - with-security
+  - name: security-analytics
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: sql
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+      additional-cluster-configs:
+        script.context.field.max_compilations_rate: 1000/1m
+        plugins.query.datasources.encryption.masterkey: 4fc8fee6a3fd7d6ca01772e5
+    bwc-test:
+      test-configs:
+        - with-security
+  - name: custom-codecs
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: skills
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: query-insights
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: opensearch-system-templates
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
```

Comment:
Hi @peterzhuamazon,
Do you know from where the data is generated. Looks like some flags are missing https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.17.0/opensearch-2.17.0-test.yml#L167-L169

cc: @vikasvb90

Reply:
Is [this](https://github.com/opensearch-project/opensearch-build/blob/d03f2977e5f75eda6949a06c40881bf9d6be73c5/scripts/deploy_single_tar_cluster.sh#L117) the place to add?

Reply:
@gaiksaya 
```

2024-10-25 00:51:25 INFO     Using /var/jenkins/workspace/manifest-update/manifests/2.18.0/opensearch-2.18.0.yml as the base input manifest
2024-10-25 00:51:25 INFO     Using /var/jenkins/workspace/manifest-update/manifests/2.18.0/opensearch-2.18.0-test.yml as the base test manifest
2024-10-25 00:51:25 INFO     Wrote /var/jenkins/workspace/manifest-update/manifests/2.19.0/opensearch-2.19.0.yml as the new input manifest
2024-10-25 00:51:25 INFO     Wrote /var/jenkins/workspace/manifest-update/manifests/2.19.0/opensearch-2.19.0-test.yml as the new test manifest
```

Reply:
So yes it is based on the 2.18.0-test manifest.

I will update both 2.18 and 2.19.